library/Fission/Email/SendInBlue/Request/Types.hs

```diff
@@ -0,0 +1,23 @@
+module Fission.Email.SendInBlue.Request.Types (Request(..)) where
+
+import           Fission.Prelude
+
+import           Fission.Email.Recipient.Types
+import           Fission.Email.SendInBlue.TemplateOptions.Types
+import           Fission.Email.SendInBlue.TemplateId.Types
+
+import qualified Data.Vector as Vector
+
+
+data Request = Request
+  { templateId :: !TemplateId
+  , to         :: ![Recipient]
```

Comment:
Can it be empty?

Reply:
:thinking: wouldn't make much sense for it to be

Reply:
I'll change it to `NonEmpty`